srcpkgs/minbif/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'minbif'
+pkgname=minbif
+version=1.0.5
+revision=1
+wrksrc="$pkgname-$version"
+build_style=cmake
+configure_args="PREFIX=/usr CONF_PREFIX=/etc/minbif ENABLE_CACA=$(vopt_if caca 'ON' 'OFF') ENABLE_VIDEO=$(vopt_if caca 'ON' 'OFF') ENABLE_PAM=ON ENABLE_TLS=ON DEBUG=OFF"
```

Comment:
Cut at the last space before hitting 80 columns.

Cmake arguments use `-D<key>=<value>` so you're missind the `-D`

Reply:
Actually the do_install use `make` which encapsulate the consigure_args without using the `-D`